.circleci/config.yml

```diff
@@ -7,11 +7,12 @@ jobs:
       - image: cypress/base:12
     steps:
       - checkout
-      - node/install-packages
+      - node/install-packages:
+          pkg-manager: yarn
       - run: npm run affected:lint -- --all
       - run: npm run affected:test -- --all
       - run: npm run affected:e2e -- --all
```

Comment:
You're telling me you want to use `yarn`, but you're not showing me you want to use it.